tst/EnergyPlus/unit/ThermalComfort.unit.cc

```diff
@@ -390,7 +390,7 @@ TEST_F( EnergyPlusFixture, ThermalComfort_CalcThermalComfortFanger )
 		" ,                         !- Lighting Level{ W }",
 		" 10.0,                     !- Watts per Zone Floor Area{ W / m2 }",
 		" ,                         !- Watts per Person{ W / person }",
-		" 0.1,                      !- Return Air Fraction",
+		" 0.0,                      !- Return Air Fraction",
```

Comment:
Is this to avoid having to change all the values being checked by the unit test?

Reply:
Return air doesn't impact thermal comfort calcs. This change was to avoid tripping on [this error](https://github.com/NREL/EnergyPlus/blob/fc306778e169208ee5d29ce9d678d1e57f13eb53/src/EnergyPlus/InternalHeatGains.cc#L1152-L1156) because the unit test doesn't have any conditioned zone info.